MainModule/Server/Commands/HeadAdmins.lua

```diff
@@ -129,7 +129,7 @@ return function(Vargs, env)
 
 		UnGameBan = {
 			Prefix = Settings.Prefix;
-			Commands = {"ungameban", "saveunban", "undataban", "unpban"};
+			Commands = {"unpermenantban", "unpermban", "unpban"};
```

Comment:
```suggestion
			Commands = {"unpermenantban", "unpermban", "unpban"};
			Commands = {"unpermenantban", "unpermban", "unpban", "ungameban", "saveunban", "undataban"};

```
This is for the same reason as the other one - prevent confusion and also prevents people getting the impression that they can no longer ungameban people.